sparse/tests/test_dot.py

```diff
@@ -316,3 +317,10 @@ def test_complex(dtype1, dtype2, format1, format2):
     x1, x2 = dense_convertor(s1), dense_convertor(s2)
 
     assert_eq(x1 @ x2, s1 @ s2)
+
+
+@given(x=st.integers(min_value=0), y=st.integers(min_value=0))
```

Comment:
Do you know the maximum value here? We don't want to create arrays that are too large.

Reply:
Will an upper limit of 100 be okay?

Reply:
10 is more than enough, I think. 😄